GVRf/Framework/jni/gl/gl_texture.h

```diff
@@ -56,8 +56,10 @@ class GLTexture : public GLPendingTask {
         memcpy(texture_parameters_, texture_parameters, sizeof(int) * 5);
     }
 
-    ~GLTexture() {
-        deleter_->queueTexture(id_);
+    virtual ~GLTexture() {
+        if (GL_TASK_NONE == pending_gl_task_) {
```

Comment:
Not sure the reason for this check. If it is to make sure id_ is valid, it should be initialized to an invalid value, and check against that.